common/exception/ErrorMessage.java

```diff
@@ -278,13 +278,13 @@ public static class Pattern extends ErrorMessage {
         public static final Pattern MULTIPLE_TYPE_CONSTRAINT_REGEX =
                 new Pattern(15, "The type variable '%s' has multiple 'regex' constraints.");
         public static final Pattern UNSATISFIABLE_PATTERN =
-                new Pattern(16, "The pattern '%s' can never be satisfied in the current schema.");
+                new Pattern(16, "The pattern '%s' is illegal in the current schema.");
```

Comment:
I find "cannot be satisfied" is too ambiguous when compared with un-answerable. Illegal is a stronger term.